nativelib/src/main/resources/gc/immix/Heap.c

```diff
@@ -129,7 +129,7 @@ NOINLINE word_t *Heap_allocSmallSlow(Heap *heap, uint32_t size) {
     if (object != NULL)
         goto done;
 
-    Heap_Grow(heap, size);
+    Heap_Grow(heap, WORDS_IN_BLOCK);
```

Comment:
Needs a comment to indicate that size is always less than WORDS_IN_BLOCK because small allocations are always lss than 8K whereas blocks are 32K.